app/components/__tests__/FormLoginButton.test.js

```diff
@@ -0,0 +1,41 @@
+// @flow
+
+import React from "react";
+import { shallow } from "enzyme";
+
+import { Button } from "native-base";
+
+import { FormLoginButtonUnwrapped } from "../FormLoginButton";
+
+describe("Loading", () => {
+  it("has a valid snapshot", () => {
+    const mutateMock = () => jest.fn();
```

Comment:
Nitpicky, but this doesn't need to be a jest fn, since you're not making any assertions on it. It could just be an empty anonymous function (`() => {}`). I think the only downside of using a jest mock is that it could prime the reader to look out for an assertion being made which never happens.